K2Bridge.Tests.UnitTests/Visitors/AggregationVisitorTests.cs

```diff
@@ -23,7 +23,7 @@ public string AggregationVisit_WithEmptyAgg_ReturnsNoPrimary()
             return aggregateClause.KustoQL;
         }
 
-        [TestCase(ExpectedResult = "_A=avg(fieldA)")]
+        [TestCase(ExpectedResult = "['_A']=avg(fieldA)")]
```

Comment:
why the [] ?

Reply:
See above: identifier quotes in Kusto allow using entity names containing a dash. This happens in some cases for the Terms aggregation.